android/src/main/java/com/unflowreactnativesdk/UnflowModule.kt

```diff
@@ -30,15 +31,20 @@ class UnflowModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     fun setAttributes(attributes: ReadableMap) {
-      val mappedAttributes = attributes as Map<String, String>
+      val mappedAttributes = attributes.toHashMap() as Map<String, String>;
```

Comment:
you can get rid of the `;` at the end

not sure you need the `as Map<String, String>` because it could be `<String, Any?>` and not just `<String, String>`?

Reply:
I thought so too but I'm getting this error when I change it to that
```
val mappedAttributes = attributes.toHashMap() as Map<String, Any?>


Type mismatch.
Required:
Map<String, String>
Found:
Map<String, Any?>
```

Reply:
oh looks like custom attributes can only be `String` values. can't remember why though... maybe because that's what the backend expects?

Reply:
It is for now, because we only have the Text attribute type implemented so far. We should expand that to an `Any` and let people send integers, date, etc

Reply:
Not a high priority yet though. Strings are a good start for anyone looking to track attributes

Reply:
yep let's keep it as it is on the client then so that people don't accidentally send the wrong thing for now 👍